content/en/blog/2023-09-12-year-one-lessons.md

```diff
@@ -0,0 +1,56 @@
+---
+title: My First Year as a Junior Developer at Rotational Labs
+slug: "year-one-lessons"
+date: "2023-08-28T19:22:43-04:00"
```

Comment:
To make this visible during the review, I didn't change the date. However, I was thinking of publishing this on 9/12 since that'll be a year from my start date.